client/environment_variables_get.go

```diff
@@ -32,3 +32,25 @@ func (c *Client) getEnvironmentVariables(ctx context.Context, projectID, teamID
 	err = c.doRequest(req, &envResponse)
 	return envResponse.Env, err
 }
+
+func (c *Client) GetEnvironmentVariable(ctx context.Context, projectID, teamID, envID string) (e EnvironmentVariable, err error) {
+	url := fmt.Sprintf("%s/v1/projects/%s/env/%s?decrypt=true", c.baseURL, projectID, envID)
```

Comment:
can we check if this needs the `decrypt=true` parameter?

Reply:
Yeah we don't need that